### PR TITLE
tools: FreeBSD basic getopt does not use --options

### PR DESCRIPTION
**Pull request by wjwithagen**
Signed-off-by: Willem Jan Withagen <wjw@digiware.nl>

### CODE REVIEW

File: src/tools/setup-virtualenv.sh [hunk 1]
```diff
@@ -17,6 +17,11 @@
 
 SCRIPTNAME="$(basename $0)"
 PYTHON_BINARY="python2.7"
+if [ `uname` == FreeBSD ]; then
+    GETOPT=/usr/local/bin/getopt
```
Comment (smithfarm) [inline src/tools/setup-virtualenv.sh]: put `/usr/local/bin/getopt` in double-quotes?

File: src/tools/setup-virtualenv.sh [hunk 2]
```diff
@@ -17,6 +17,11 @@
...
+else
+    GETOPT=getopt
```
Comment (smithfarm) [inline src/tools/setup-virtualenv.sh]: put `getopt` in double-quotes? I know it's not striclty necessary - just to unify the coding style.

Comment (wjwithagen) [reply src/tools/setup-virtualenv.sh]: @smithfarm 
Sure, no problem.

**Comment by smithfarm** [comment]
retest this please

**Comment by tchaikov** [comment]
retest this please.

**Comment by wjwithagen** [comment]
@smithfarm @tchaikov 
Why would this be incorrectly signed off??
Will run it thru amend and see if if fixes anything.

**Comment by tchaikov** [comment]
retest this please.

**Comment by smithfarm** [comment]
Failed run-rbd-unit-tests.sh

Failed unittest_journal (crashes on assert(!m_metadata->is_initialized())

**Comment by smithfarm** [comment]
retest this please

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2018-04-04_00:32:09-rados-wip-kefu-testing-2018-04-04-0003-distro-basic-mira/


@smithfarm this PR compiles. so i guess it's good to merge, am i right?

**Pull request merged**